PopupBridgeExample/src/androidTest/java/com.braintreepayments.popupbridge.example.test/PopupBridgeTest.java

```diff
@@ -46,7 +48,8 @@ public void opensPopup_whenClickingDontLikeAnyOfTheseColors_returnsCanceledSelec
         onViewWithText("Launch Popup").perform(click());
         onViewWithText("I don't like any of these colors")
                 .waitForExists(BROWSER_TIMEOUT).perform(click());
-        onViewWithText("You do not like any of these colors").waitForExists();
```

Comment:
This test was previously passing even though the text did not match what was on the screen (these vs. our). Converting the test to use `containsString()` caused the test to fail as expected with non-matching text, and pass with matching text on screen.